variables.tf

```diff
@@ -1,6 +1,6 @@
 variable "alt_hostname" {
   description = "Specify an alternative hostname for the public website url (instead of public_zone_name)"
-  type    = map(object({
+  type    = map(object({https://github.com/avalonmediasystem/avalon-terraform
```

Comment:
I don't understand why the repo url is added here.  Is this a typo or does it serve some purpose?

Reply:
@cjcolvar: That's definitely a typo, likely an accidental middle-button paste. I'll update my pull request.